### DISCUSSION TITLE
Is there a way to filter nations to a subset?

### DISCUSSION THREAD

**Discussion by jimlad999** [Luke100000/minecraft-comes-alive#738] [Q&A] [answered]
Hi,

I see you have a config option for selecting just USA names, `useModernUSANamesOnly`.
https://github.com/Luke100000/minecraft-comes-alive/blob/1.19.2/common/src/main/java/net/mca/resources/Names.java#L52-L59

I was curious if there was a way to filter to a subset instead?
e.g. take only names from region names "finland", "korea", "modernusa"?

Kind regards,
jim

**Comment by Luke100000** [comment, marked as answer]
No, and since datapacks can't delete content afaik the only solution is to modify the jar directly. Those are the files: https://github.com/Luke100000/minecraft-comes-alive/tree/1.19.2/common/src/main/resources/data/mca/names

**Reply by jimlad999** [reply]
Would you be open to a feature request?
e.g.
In the config mca.json have an array of strings for regions along side `useModernUSANamesOnly`?
```
"useModernUSANamesOnly": false,
"regionsNamesWhitelist": [ "finland", "korea", "modernusa" ],
```

Or is that not inline with the direction you would want to take this mod?

Kind regards,
jim
